app/views/govuk_publishing_components/component_guide/show.html.erb

```diff
@@ -2,6 +2,18 @@
 <%= render 'govuk_component/title', title: @component_doc.name, context: 'Component' %>
 
 <div class="component-show">
+
+  <% if !@component_doc.accessibility_criteria.present? %>
+    <div class="grid-row">
+      <div class="column-two-thirds">
```

Comment:
Is there a reason why we are using this class - we could just remove this and make the warning banner full width to fit with our other warning/violation styles?

Reply:
@fofr - can you advise? I put this in because the following grid row (the component description) has it. When I strip it out from both the description stays at less than full width because of other styles applied to it. It looks a bit odd if the warning is wide and the description is narrow: 

![screen shot 2017-10-12 at 10 47 40](https://user-images.githubusercontent.com/31649453/31490144-e3f0b7b4-af3a-11e7-8e75-40540539ea81.png)

Reply:
I'd suggest moving the notice into the existing grid row and cell used by the description.